.gitignore

```diff
@@ -36,4 +36,7 @@ npm-debug.log
 /assets/node_modules/
 
 # IDE files
-.idea
\ No newline at end of file
+.idea
+
+# Mac OS X files
+.DS_Store
```

Comment:
Lets add a new line
```suggestion
.DS_Store

```

Reply:
Thanks. For this project, I decided to try VS code (previously I worked with Intellij IDEA), and it turned out that in VS code option "Insert new line on file save" is disabled by default, don't know why they have such a default.

I have enabled this option in IDE and fixed the file as you suggested.